test/check-machines-disks

```diff
@@ -553,8 +551,7 @@ class TestMachinesDisks(VirtualMachinesCase):
                                                     ignore=[self.pixel_test_ignore] if self.pixel_test_ignore else [],
                                                     skip_layouts=["rtl"])
 
-            if not self.skip_add:
-                self.add_disk()
```

Comment:
Why was this dropped?

Reply:
See commit message:
```
    Two small behavior changes were introduced:
    First, on the validation of the input form. Previously the 'custom path'
    error was introduced also before 'add' was clicked. Now, this behaves like all other form fields,
    and validates on user's first 'Add' click.
```

So skip_add is not needed anymore, all the validation happens after the user clicks "Add" for first time. This change I did for consistently handling all verification errors and not have special cases there. I can revert if needed.

Reply:
Perfect, thanks! No no, I'm all for removing special cases, I was just wondering.